src/main/java/com/hillel/dzhurynskyi/homework/hm6/ArrayValueCalculator.java

```diff
@@ -0,0 +1,29 @@
+package com.hillel.dzhurynskyi.homework.hm6;
+
+public class ArrayValueCalculator {
+
+    public int doCalc(String [][]arr) throws ArraySizeException, ArrayDataException{
+        int suma = 0;
+
+        if (arr.length > 4) {
+            throw new ArraySizeException();
+        } else {
+            for (int i = 0; i < 4; i++) {
+                if (arr[i].length > 4) {
+                    throw new ArraySizeException("index "+i);
+                } else {
+                    for (int j = 0; j < 4; j++) {
+                        System.out.print(arr[i][j] + " ");
```

Comment:
If we will have an arr[i].length < 4, what surprise will happen in arr[i][j]? j=0..3